openlibrary/core/sponsorships.py

```diff
@@ -45,25 +45,7 @@ def get_sponsored_editions(user):
     contact_id = get_contact_id_by_username(archive_id)
     return get_sponsorships_by_contact_id(contact_id) if contact_id else []
 
-def get_sponsorable_editions():
-    """This should move to an infogami type so any admin can add editions
-    to the list. This will need to be paginated.
-    """
-    try:
-        candidates = web.ctx.site.get('/sponsorship/books').get('editions', [])
-    except AttributeError:
-        candidates = []
-
-    eligible_editions = []
-    for key in candidates:
-        ed = web.ctx.site.get(key)
-        ed.eligibility = qualifies_for_sponsorship(ed)
-        if ed.eligibility.get('is_eligible'):
-            eligible_editions.append(ed)
-    return eligible_editions
-
-
-def do_we_want_it(isbn, work_id):
+def do_we_want_it(isbn, work_id=None):
```

Comment:
Guessing `work_id` was left in case we want to re-enable the check in the future? If so, let's remove it; this is called only once in the code, and it doesn't specify a `work_id`. If we need to reinstate, that's what git history is for.

Reply:
:+1: